language-extensions/fhir-extension/src/main/java/dev/ikm/tinkar/fhir/transformers/FhirUtils.java

```diff
@@ -35,15 +35,15 @@
 import java.util.Map;
 import java.util.function.BiConsumer;
 
-import static dev.ikm.tinkar.fhir.transformers.FhirConstants.ROLE_GROUP_URL;
-import static dev.ikm.tinkar.fhir.transformers.FhirConstants.SNOMEDCT_URL;
+import static dev.ikm.tinkar.fhir.transformers.FhirConstants.*;
```

Comment:
Let's not use wildcards on imports. IntelliJ can manage this for you. You can go to settings -> Editor -> Code Style -> Java -> Imports and then set "Class count to use import with '*' and "Name count to use static import '*'" both to something high like 100.